scripts/mk.js

```diff
@@ -18324,132 +18326,207 @@ var clLocalVars, clHud, clSelected;
-		dance: for (var i=0;i<oItemDistributions.length;i++) {
-			var oItemDistribution = oItemDistributions[i];
-			for (var j=0;j<oItemDistribution.value.length;j++) {
-				if (oItemDistribution.value[j][wObject] != null) {
-					isExistingObj = true;
-					break dance;
-				}
-			}
-		}
-		if (!isExistingObj)
-			return false;
-		if (oPlayer.billball) {
-			oPlayer.stash = wObject;
-			oPlayer.roulette2 = 25;
-		}
-		else {
-			oPlayer.arme = wObject;
-			oPlayer.roulette = 25;
-		}
-		updateObjHud(0);
-		return true;
-	}
-	var isTP = /^tp ([\d\-+\.]+) ([\d\-+\.]+)$/g.exec(cheatCode);
-	if (!isTP)
-		isTP = /^tp ([\d\-+\.]+) ([\d\-+\.]+) ([\d\-+\.]+)$/g.exec(cheatCode);
-	if (isTP) {
-		var x = parseFloat(isTP[1]), y = parseFloat(isTP[2]);
-		if (isNaN(x) || isNaN(y))
-			return false;
-		var th = parseFloat(isTP[3]);
-		oPlayer.x = x;
-		oPlayer.y = y;
-		if (!isNaN(th))
-			oPlayer.rotation = th;
-		resetRenderState();
-		return true;
-	}
-	var isLap = /^lap(?: ([1-3]|c))?(?: (\d+|c))?$/g.exec(cheatCode);
-	if (isLap) {
-		var t = parseInt(isLap[1]), c = parseInt(isLap[2]);
-		if (isLap[1] == "c")
-			t = oPlayer.tours;
-		if (!isLap[1]) {
-			t = oMap.tours;
-			var lMap = getCurrentLMap({ tours: t, demitours: 0 });
-			c = lMap.checkpoint.length;
-		}
-		if (c > 0) c--;
-		if (isLap[2] == "c")
-			c = oPlayer.demitours;
-		if (!isLap[2]) {
-			var lMap = getCurrentLMap(getCurrentLapId(oPlayer));
-			c = lMap.checkpoint.length-1;
-		}
-		if (isNaN(t) || isNaN(c))
-			return false;
-		var prevLap = oPlayer.tours, prevCP = oPlayer.demitours;
-		oPlayer.tours = t;
-		oPlayer.demitours = c;
-		var lMap = getCurrentLMap(oPlayer);
-		var checkpoint = lMap.checkpointCoords[c];
-		if (checkpoint && cheatCode !== 'lap') {
-			var nextCp = lMap.checkpointCoords[(c+1)%lMap.checkpointCoords.length];
-			oPlayer.x = checkpoint.O[0];
-			oPlayer.y = checkpoint.O[1];
-			oPlayer.rotation = Math.atan2(nextCp.O[0]-checkpoint.O[0],nextCp.O[1]-checkpoint.O[1])*180/Math.PI;
-		}
-		updateLapHud(0);
-		handleCpChange(prevLap,prevCP, 0);
-		return true;
-	}
-	if (cheatCode == "pos") {
-		alert("x: "+ Math.round(oPlayer.x)+"\ny: "+Math.round(oPlayer.y)+"\ntheta: "+Math.round(oPlayer.rotation));
-		return true;
-	}
-	else if (cheatCode == "xs") {
-		oPlayer.size = 0.6;
-		oPlayer.mini = 0;
-		return true;
-	}
-	else if (cheatCode == "md") {
-		oPlayer.size = 1;
-		oPlayer.mini = 0;
-		return true;
-	}
-	else if (cheatCode == "xl") {
-		oPlayer.size = Math.pow(1.05,8);
-		oPlayer.mini = 0;
-		return true;
-	}
-	if (course == "BB") {
-		if (cheatCode == "balloon")
-			cheatCode += " 1";
-		var isBaloon = /^balloon (\d+)$/g.exec(cheatCode);
-		if (isBaloon) {
-			var toAdd = parseInt(isBaloon[1]);
-			if (toAdd) {
+    const rawArgs = cheatArgs(cheatCode);
+    if (!rawArgs) return false;
+
+    const command = rawArgs[0];
+    const args = rawArgs.slice(1);
+    const oPlayer = oPlayers[0];
+
+    switch (command) {
+		case "arme":
```

Comment:
It's not exactly what i had in mind, I imagined `/give` to change the 1st item and `/stash` to change the 2nd one. We can keep this behavior if you prefer but `/arme` is a bit confusing (even in French 😆), maybe find an other name? Idk `/slot`? 🤷

Reply:
Other option, we can also keep the idea of a boolean parameter but in a less confusing way, like `/give xxx 1` => adds to the 1st slot, `/give xxx 2` => add to the 2nd slot. The parameter would be optional, `/give xxx` does what you currently implemented